src/xo-app/vm/tab-advanced.js

```diff
@@ -574,6 +612,14 @@ export default connectStore(() => {
                   : _('unknownOriginalTemplate')}
               </td>
             </tr>
+            {isAdmin && (
+              <tr>
+                <th>{_('resourceSet')}</th>
+                <td>
+                  <VmResourceSet vm={vm} />
+                </td>
+              </tr>
+            )}
```

Comment:
Maybe we should display the resource set for non admins, but it definitely should not be editable (this should be enforced server-side).